<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDDeviceConfigDialog</class>
 <widget class="QDialog" name="CMDDeviceConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>520</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>491</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>通信参数</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>通信模式</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>481</width>
       <height>191</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
      <widget class="QComboBox" name="portComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="baudRateComboBox">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="parityComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>181</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ODD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EVEN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MARK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SPACE</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="stopBitsComboBox">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="dataBitsComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>主站IP地址</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverPortEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>主站服务端口</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverIPEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>从站IP地址</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverSlaveIPEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>从站服务端口</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="serverSlavePortEdit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>通信参数</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usbCommParameterEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLineEdit" name="boardCommParameterEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>通信参数</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLineEdit" name="otherCommParameterEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>431</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>通信参数</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="communicationTypeBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UDP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Board</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
    <zorder>stackedWidget</zorder>
    <zorder>label_3</zorder>
    <zorder>communicationTypeBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>设备参数</string>
    </attribute>
    <widget class="QLineEdit" name="replyTimeoutEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>命令超时(MS)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="protocolEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Secp TCP</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>通信协议</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="retriesEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>命令重试次数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="configButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>获取配置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="statuEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>260</y>
       <width>351</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>获取配置状态显示</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
